### DISCUSSION TITLE
第6节课笔记(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#455] [笔记]
请大家把第六节课笔记放在这里哦~

**Comment by zhjunqin** [comment]
zhjunqin: https://blog.csdn.net/zhjunqin/article/details/135727095?spm=1001.2014.3001.5502

**Comment by onlyzer0** [comment]
onlyzer0:https://blog.csdn.net/qq_42907812/article/details/135758139?spm=1001.2014.3001.5502

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/42313ac0d8084589b5d84d853a5b3378?pvs=4
